aws/product/awsec2product.go

```diff
@@ -0,0 +1,191 @@
+package product
+
+import (
+	"encoding/json"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"strconv"
+
+	"github.com/trackit/jsonlog"
+	"github.com/trackit/trackit2/db"
+	"github.com/trackit/trackit2/models"
```

Comment:
Please respect organize your imports in three sections : standard library imports, imports from other projects (not `github.com/trackit/trackit2`) and imports from the current project.